src/qibocal/protocols/dispersive_shift.py

```diff
@@ -289,54 +295,51 @@ def _plot(data: DispersiveShiftData, target: QubitId, fit: DispersiveShiftResult
                 col=1,
             )
 
-    if fit is not None:
-        fig.add_trace(
-            go.Scatter(
-                x=[
-                    fit.best_freq[target] * HZ_TO_GHZ,
-                    fit.best_freq[target] * HZ_TO_GHZ,
-                ],
-                y=[
-                    np.min(np.concatenate((data_0.signal, data_1.signal))),
-                    np.max(np.concatenate((data_0.signal, data_1.signal))),
-                ],
-                mode="lines",
-                line=go.scatter.Line(color="orange", width=3, dash="dash"),
-                name="Best frequency",
-            ),
-            row=1,
-            col=1,
-        )
+    if fit_data_0 != None or fit_data_1 != None:
```

Comment:
```suggestion
    if fit_data_0 is not None or fit_data_1 is not None:
```

`None` is a unique value in its type, so you could check for identity, rather than equality (it doesn't matter that match, but it's more idiomatic).